exodus_gw/publish.py

```diff
@@ -0,0 +1,9 @@
+import uuid
+
+
+def create_publish_id():
+    """Create a new, empty publish id"""
+
+    publish_id = uuid.uuid4()
+
+    return {"detail": publish_id}
```

Comment:
Sorry, I still don't understand the reason to return a dictionary.
This is just a backend function, right? So, it can just return `publish_id` to the caller, which, right now, is the publish endpoint.
```suggestion
    return publish_id
```
The caller can then expose to the user whatever details it needs to in the forthcoming response object.

Reply:
+1

Reply:
See my previous comment